client/sources/ok_test/models.py

```diff
@@ -100,9 +100,11 @@ def unlock(self, interact):
                                 for case in suite.cases])
         for suite_num, suite in enumerate(self.suites):
             for case_num, case in enumerate(suite.cases):
+                case_id = '{} > Suite {} > Case {}'.format(
```

Comment:
Ah, I see. By "question", I thought you actually meant "suite".

We should change 'question' to 'prompt' then.

Those two things would have different suite texts since the first one would be x = 5 and the second would be x = -5. And even if the prompt was the same twice (e.g. if someone, for some reason, did '>>> square(x)' twice in the first example) then the entire suite would have the same answer locked/unlocked depending on what the student was answering.